jscomp/gentype_tests/typescript-react-example/src/BucklescriptAnnotations.res.js

```diff
@@ -1,9 +1,10 @@
 // Generated by ReScript, PLEASE EDIT WITH CARE
 
+import * as Curry from "rescript/lib/es6/curry.js";
 
 function bar(x) {
   let f = x.twoArgs;
-  return f(3, "a");
+  return Curry._2(f, 3, "a");
```

Comment:
Now a few Curry calls are back, e.g. here

Reply:
This is the PR that introduces curry.
The one that removes it is https://github.com/rescript-lang/rescript-compiler/pull/6894